### PR TITLE
Adding VD to Port and Azure SP Connection

### PR DESCRIPTION
**Pull request by jkallem-equinix**

### CODE REVIEW

File: .github/workflows/terratests-prod-suite.yaml [hunk 1]
```diff
@@ -64,6 +66,9 @@ jobs:
           echo $TEST_DATA_PROD_CLOUD_ROUTER_2_SERVICE_PROFILE_CONNECTION >> "./examples/cloud-router-2-service-profile-connection/terraform.tfvars.json"
           echo $TEST_DATA_PROD_CLOUD_ROUTER_2_WAN_CONNECTION >> "./examples/cloud-router-2-wan-connection/terraform.tfvars.json"
           echo $TEST_DATA_PROD_VIRTUAL_DEVICE_2_WAN_CONNECTION >> "./tests/examples-without-external-providers/virtual-device-2-wan-connection/terraform.tfvars.json"
+          echo $TEST_DATA_PROD_VIRTUAL_DEVICE_2_AZURE_CONNECTION >> "./tests/examples-without-external-providers/virtual-device-2-azure-connection/terraform.tfvars.json"
+          echo $TEST_DATA_PROD_VIRTUAL_DEVICE_2_PORT_CONNECTION >> "./tests/examples-without-external-providers/virtual-device-2-port-connection/terraform.tfvars.json"
+
```
Comment (srushti-patl) [inline .github/workflows/terratests-prod-suite.yaml:71]: Remove extra space

File: .github/workflows/terratests-prod-suite.yaml
Comment (srushti-patl) [inline .github/workflows/terratests-prod-suite.yaml:1]: Add screenshot of passing PROD Sanity GHA workflow

**Comment by jkallem-equinix** [comment]
Screenshot 2024-04-16 at 5 29 46 PM

**Review by srushti-patl — APPROVED** [review]
LGTM

**Pull request merged**